packages/moonstone/Scrollable/ScrollableNative.js

```diff
@@ -560,6 +570,60 @@ class ScrollableBaseNative extends Component {
 		}
 	}
 
+	updateFocusAfterVoiceControl = () => {
+		const
+			{direction} = this.props,
+			viewportBounds = this.uiRef.containerRef.getBoundingClientRect(),
+			spotItemBounds = document.activeElement.getBoundingClientRect(),
+			nodes = this.uiRef.containerRef.querySelectorAll('[data-index]'),
+			isVertical = (direction === 'vertical' || direction === 'both'),
+			first = isVertical ? 'top' : 'left',
+			last = isVertical ? 'bottom' : 'right';
```

Comment:
Do you intend to scroll only vertically even though `direction` is equal to `both`? If yes, no problem. I just wanted to check if it was intended or not.

Reply:
If no, VirtualList supports `topleft` and `bottomright` option with `align` property.

Reply:
In case of both, it operates only horizontally and vertically.(topleft and bottomright is not supported by voice control: http://collab.lge.com/main/pages/viewpage.action?pageId=840355974&focusedCommentId=853653402#comment-853653402 ) Relate to this, change to check the voice control direction using by seperate variable(voiceControlDirection)